_pages/about.md

```diff
@@ -7,13 +7,17 @@ permalink: /about
 ---
 
 ## What is this
-As it says up there, this is a site dedicated to providing sane, informed written criticism of the latest AI hype and doomsaying. In other words, the idea is to post a concise but comprehensive overview of each major AI story that bubbles up in popular media. Such stories have happened more and more as the AI hype wave has gone on, to the point where it seems something like this needs to exist. 
+
+As it says up there, this is a site dedicated to providing sane, informed written criticism of the latest AI hype and doomsaying. In other words, the idea is to post a concise but comprehensive overview of each major AI story that bubbles up in popular media. Such stories have happened more and more in the past several years, to the point where it seems something like this needs to exist. The hope is to get a range of people in CS/AI to cover the coverage of new research/events and so provide make it clear to people less informed about AI what is misinformation and what is actually worth paying attention to. 
 
 ## Who are you
-We are a group of people with education in Computer Science and various levels of domain knowledge in AI. The chief editor and creator of the publication is [Andrey Kurenkov](http://www.andreykurenkov.com/), who is a graduate student at Stanford and doing research in AI and has [written](http://www.andreykurenkov.com/writing/ai/a-brief-history-of-neural-nets-and-deep-learning/) [several](http://www.andreykurenkov.com/writing/ai/a-brief-history-of-game-ai/) long-form explanatory pieces about AI in the past. 
+We are a group of researchers and engineers with domain knowledge in AI collaborating to this site's effort in our free time. People actively contributing to the site include:
```

Comment:
This is worded super awkwardly. Maybe "with domain knowledge in AI who collaborate toward this goal in our spare time" or something?

Reply:
Good catch